lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -542,7 +542,7 @@ def rescore_problem_module_state(xmodule_instance_args, module_descriptor, stude
             if not instance.done:
                 return UPDATE_STATUS_SKIPPED
         else:
-            if instance.has_submitted_answer():
+            if not instance.has_submitted_answer():
```

Comment:
Cool.  Seems sensible.  Can you make this an elif instead (nit), and make sure it's caught by a test (non-nit).